free-courses-pt_BR.md

```diff
@@ -153,6 +160,7 @@
 ### Node.js
 
 * [Criando APIs com NodeJs ](https://www.youtube.com/playlist?list=PLHlHvK2lnJndvvycjBqQAbgEDqXxKLoqn) - Balta.io (YouTube)
+* [Node.js para iniciantes](https://treinamento.nodebr.org) - NodeBr - Erick Wendel
```

Comment:
requires email/signup

Reply:
please review our submission guidelines for resources that require email or registration to access.